src/providers/contacts/hooks.ts

```diff
@@ -0,0 +1,4 @@
+import { useContext } from 'react'
+import { ContactsContext } from './provider'
+
+export const useContacts = () => useContext(ContactsContext)
```

Comment:
Hook names must start with `use` followed by a capital letter. 
This convention guarantees that you can always look at a component and know where its state, Effects, and other React features might “hide”.

Here we don't have state, effects or calls of other hooks. It should be converted to a regular function. Also it will brake hooks rules.

Reply:
Totally don't get why you inform me about that...

It is a custom hook which wraps useContext. Show me the rule I'm breaking here...

Reply:
Will do it 👍